src/app/Http/Controllers/Api/MovieController.php

```diff
@@ -0,0 +1,77 @@
+<?php
+
+namespace App\Http\Controllers\Api;
+
+use App\Http\Controllers\Controller;
+use App\Models\Movie\Movie;
+use Illuminate\Http\Request;
+
+class MovieController extends Controller
+{
+    /**
+     * Display a listing of the resource.
+     *ß
+     * @return \Illuminate\Http\Response
+     */
+    public function index()
+    {
+        return response(Movie::all(), 200);
```

Comment:
https://laravel.com/docs/9.x/eloquent-resources#resource-collections
Check this out